communitiesuk/funding-service-design-frontend
# FS-3428 Adjusting all questions route to deal with language code in f…

### Change description
Updates the all questions route to deal with the language code being in the filename for all welsh funds after COF R3.

The existing method of having a welsh version of the all questions page is to have translation tags in the template, and call the correct string from babel at runtime.
Now the all questions pages are generated from the forms, we just generate a page for english and a page for welsh, so this method determines which template to render based on language code.



### How to test
Visit the all questions page for all exisiting funds/rounds (COF, NSTF, DPIF, CYP) and check it renders in the expected language
